spark-cluster/src/main/scala/tech/ytsaurus/spark/launcher/SparkLauncher.scala

```diff
@@ -195,7 +194,7 @@ trait SparkLauncher {
     val startProcessCode = startProcess.exitValue()
     log.info(f"Server started. Code: $startProcessCode")
     if (startProcessCode == 0) {
-      val pid = (f"cat $livyHome/livy--server.pid".!!).trim
+      val pid = Seq("bash", "-c", """cat "$0"/livy-*-server.pid""", livyHome).!!.trim
```

Comment:
It was `livy-root-server.pid`:
https://github.com/apache/incubator-livy/blob/6097af1cdd536ebbe1d7eacb1513a440a5fa2784/bin/livy-server#L46